src/django/api/templates/mail/report_result_body.html

```diff
@@ -10,22 +10,32 @@
             Hi there,
         </p>
         <p>
-            Thanks for reporting {{ facility_name }} facility as {{ closure_state|lower }}.
-            {% if is_closure and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_closure and is_confirmed %}The facility profile in the Open Supply Hub has now been updated and the facility is marked as closed.{% endif %}{% if is_reopening and is_rejected %}We have rejected this report and the facility's profile in OS Hub has not been changed.{% endif %}{% if is_reopening and is_confirmed %}The facility profile in the Open Supply Hub has now been updated and the facility is no longer marked as closed.{% endif %}
+            Thanks for reporting {{ facility_name }} facility as {{
+            closure_state|lower }}. {% if is_closure and is_rejected %}We have
+            rejected this report and the facility's profile in OS Hub has not
+            been changed.{% endif %}{% if is_closure and is_confirmed %}The
+            facility profile in Open Supply Hub has now been updated and the
+            facility is marked as closed.{% endif %}{% if is_reopening and
+            is_rejected %}We have rejected this report and the facility's
+            profile in OS Hub has not been changed.{% endif %}{% if is_reopening
+            and is_confirmed %}The facility profile in Open Supply Hub has now
+            been updated and the facility is no longer marked as closed.{% endif
+            %}
         </p>
         {% if status_change_reason|length %}
         <p>
             OS Hub Team review notes: {{ status_change_reason }}
         </p>
-        {% endif %}
-        {% if is_rejected %}
+        {% endif %} {% if is_rejected %}
```

Comment:
Is there a design reason for moving the `endif` and the following `if` onto the same line?

Reply:
Over-enthusiastic formatter strikes again - I've reverted this